filament/backend/src/metal/MetalHandles.mm

```diff
@@ -277,16 +290,27 @@ static void maybePresentAndDestroyAsync(PresentDrawableData* that, bool shouldPr
     }
 
 private:
-    PresentDrawableData(id<CAMetalDrawable> drawable, MetalDriver* driver)
-        : mDrawable(drawable), mDriver(driver) {}
+    PresentDrawableData(id<CAMetalDrawable> drawable, std::mutex* drawableMutex,
+            MetalDriver* driver)
+        : mDrawable(drawable), mDrawableMutex(drawableMutex), mDriver(driver) {}
 
     static void cleanupAndDestroy(PresentDrawableData *that) {
+        std::optional<std::lock_guard<std::mutex>> opt_lock;
```

Comment:
wrong naming convention, use `optionalLock`

Reply:
Done. (If we're not making this an optional feature, then the optional isn't even needed.)